mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -296,6 +296,7 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 			ARG 1 predicate
 		METHOD method_26250 air ()Lnet/minecraft/class_4970$class_2251;
 		METHOD method_29292 requiresTool ()Lnet/minecraft/class_4970$class_2251;
+		METHOD method_31710 materialColor (Lnet/minecraft/class_3620;)Lnet/minecraft/class_4970$class_2251;
```

Comment:
```suggestion
		METHOD method_31710 mapColor (Lnet/minecraft/class_3620;)Lnet/minecraft/class_4970$class_2251;
```

Reply:
Fixed in c370022e4da786882264e042b878a59d2f569d2a